packages/api/src/external/fhir/document/index.ts

```diff
@@ -144,3 +156,110 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
     use: id.use === "unspecified" ? undefined : id.use,
   };
 }
+
+/**
+ * Takes a CW resource and converts it to a FHIR resource
+ *
+ * @param resource
+ * @returns
+ */
+function convertToFHIRResource(resource: Contained): Resource | undefined {
+  if (resource.resourceType === ResourceType.Patient && resource.id) {
+    return {
+      resourceType: ResourceType.Patient,
+      id: resource.id,
+    };
+  } else if (resource.resourceType === ResourceType.Organization && resource.name) {
+    return {
+      resourceType: ResourceType.Organization,
+      name: convertCWNameToString(resource.name),
+    };
+  } else if (resource.resourceType === ResourceType.Practitioner && resource.name) {
+    return {
+      resourceType: ResourceType.Practitioner,
+      name: convertCWNameToHumanName(resource.name),
```

Comment:
FHIR Practitioner's name is also an array of `HumanName`

Reply:
convertCWNameToHumanName returns an array of HumanName

Reply:
sounds good, I think I assumed `getHumanNamesFromObject` couldn't return an array since it gets a single object, but let's leave that for the respective thread. 👍